.github/workflows/pytest.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Upgrade pip
-      run: python -m pip install --upgrade pip
+      run: python -m pip install pip==22.0.4
```

Comment:
I'll assume this is necessary, but we should remove it later when whatever issue has been resolved.